src/js/components/organisms/hero-homepage/helpers/build-srcset.ts

```diff
@@ -0,0 +1,14 @@
+const SCREEN_WIDTHS: number[] = [640, 750, 828, 1125, 1242];
```

Comment:
Are these not defined elsewhere?

Reply:
Not for now. We need to come up with some sensible values that work for this full width component.

For other components that only show images at for example 200px wide different values are needed.